test/integration/040_override_database_test/test_override_database.py

```diff
@@ -12,6 +14,45 @@ def schema(self):
     def models(self):
         return "test/integration/040_override_database_test/models"
 
+    @property
+    def alternative_database(self):
+        if self.adapter_type == 'snowflake':
+            return os.getenv('SNOWFLAKE_TEST_DATABASE')
+        else:
+            return super(BaseOverrideDatabase, self).alternative_database
+
+    def snowflake_profile(self):
+        return {
+            'config': {
+                'send_anonymous_usage_stats': False
+            },
+            'test': {
+                'outputs': {
+                    'default2': {
+                        'type': 'snowflake',
+                        'threads': 4,
+                        'account': os.getenv('SNOWFLAKE_TEST_ACCOUNT'),
+                        'user': os.getenv('SNOWFLAKE_TEST_USER'),
+                        'password': os.getenv('SNOWFLAKE_TEST_PASSWORD'),
+                        'database': os.getenv('SNOWFLAKE_TEST_QUOTED_DATABASE'),
```

Comment:
Can you update https://github.com/fishtown-analytics/dbt/blob/dev/wilt-chamberlain/test.env.sample accordingly?